wheel/pep425tags.py

```diff
@@ -126,8 +141,23 @@ def get_platform(archive_root):
                     if version is not None:
                         base_version = max(base_version, version)
         if base_version[-1] == 0:
-            base_version = base_version[:-1]
-        base_version = "_".join([str(x) for x in base_version])
+            fin_base_version = base_version[:-1]
```

Comment:
Looks like `fin_base_version` is undefined if the if-condition is not true. This would break the code below, right? Is there any reason why you try to support x.y.z instead of just x.y?

Reply:
I use x.y.z because this is format used inside libraries header. This is 190 line of file `lib_file_analyse.py` 
```
uint32_t	version;	/* X.Y.Z is encoded in nibbles xxxx.yy.zz */
```
I push fix for undefined variable.